src/style.css

```diff
@@ -1,3 +1,54 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav {
+  display: flex;
+  gap: 20px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For instance, you should have a generic family name as a fallback, like `sans-serif`.